### DISCUSSION TITLE
Want to check if we can get access to github copilot enterprise version as an individual

### DISCUSSION THREAD

**Discussion by ghost** [community/community#130104] [Enterprise]
### Body

Hi Team, I want to know if as an individual can I get access to github copilot enterprise version for exploring its features. If yes, could you please guide on the same.

**Comment by queenofcorgis** [comment]
Hi @a-kailas-patil, 

Could you share a bit more about your use case and if you are part of an Enterprise?

**Reply by ghost** [reply]
we want to explore the enterprise feature and want to purchase a license for training purpose . please guide us

**Reply by queenofcorgis** [reply]
To explore our [Enterprise Cloud plan](https://docs.github.com/en/enterprise-cloud@latest/admin/overview/setting-up-a-trial-of-github-enterprise-cloud) there is a trial option available. You are not able to use GitHub Copilot Enterprise on this trial and GitHub Copilot Enterprise can only be used on a GitHub Enterprise Cloud account. 

If you already have GitHub Enterprise Cloud, you are able to purchase GitHub Copilot Enterprise on a per user basis. Our [docs](https://docs.github.com/en/copilot/github-copilot-enterprise/about-github-copilot-enterprise) have you covered here with all the information needed.
